WireDomain/Sources/WireDomain/Repositories/FeatureConfigRepository.swift

```diff
@@ -233,94 +230,100 @@ final class FeatureConfigRepository: FeatureConfigRepositoryProtocol {
             )
 
         case .unknown(let featureName):
-            assertionFailure("Unknown feature: \(featureName)")
+            logger.info(
```

Comment:
```suggestion
            logger.warn(
```
as it should not happen right